general/server/live-tv-setup.md

```diff
@@ -0,0 +1,85 @@
+---
+uid: server-live-tv-setup
+title: Live TV Setup Guide
+---
+
+## Add a TV tuner to Jellyfin (Automatic Discovery)

```

Comment:
For this one and all others, we should move this "one level up" so `#`
Since this will end up being the top title on the generated page., I know the other pages don't do this, but those also need changes. I just couldn't be bothered when converting from mkdocs to docfx.